.github/ISSUE_TEMPLATE/redirect-request.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Redirect Request
+about: Submit a request to implement a redirect for a URL
+title: Redirect Request
+labels: content-ia-team, ia
+assignees: mnorthuis
+
+---
+
+### Instructions
+- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
+- Please add your team's label to this issue and complete the information below.
+- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple tickets to track differnt steps.
+- It is the responsibility of the requesting team to notify VA stakeholders of pending redirect.  
+
+### Description
+*This is a detailed description of the request. It should include background information and reason for the request.*
+
+### Implementation Date
+Date new URL(s) will be live: *enter date new pages/URLs will be live*
+
+Please indicate when the redirect(s) needs to be implemented"
+- [ ] On the same date the new URLs launch
+- [ ] Can happen within 1-5 business days after new URLs launch
+- [ ] Other - Please indicate timing : 
+
+**The requesting team is responsible for notifying the group working on this issue if the target date changes**
+
+### Redirects Needed
+
+Current URL  |  Redirect to new URL
+---  |  ---
+*enter current URL* | *enter new URL*
+
+
+### Steps to complete
+- [ ] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
```

Comment:
"Above information is provided, issue is tagged with your team label, and assigned to @mnorthuis"